atom/common/crash_reporter/crash_reporter_mac.mm

```diff
@@ -4,20 +4,44 @@
 
 #include "atom/common/crash_reporter/crash_reporter_mac.h"
 
+#include "base/files/file_path.h"
+#include "base/mac/bundle_locations.h"
 #include "base/mac/mac_util.h"
 #include "base/memory/singleton.h"
+#include "base/strings/string_piece.h"
+#include "base/strings/stringprintf.h"
 #include "base/strings/sys_string_conversions.h"
-#import "vendor/breakpad/src/client/apple/Framework/BreakpadDefines.h"
+#include "vendor/crashpad/client/crash_report_database.h"
+#include "vendor/crashpad/client/crashpad_client.h"
+#include "vendor/crashpad/client/crashpad_info.h"
+#include "vendor/crashpad/client/settings.h"
+#include "vendor/crashpad/client/simple_string_dictionary.h"
 
 namespace crash_reporter {
 
-CrashReporterMac::CrashReporterMac()
-    : breakpad_(NULL) {
+namespace {
+
+crashpad::SimpleStringDictionary* g_simple_string_dictionary;
+crashpad::CrashReportDatabase* g_database;
```

Comment:
I think we can make them as members of `CrashReporterMac` instead of global variables, the usages of global variables should be as minimal as possible.